curvefs/proto/topology.proto

```diff
@@ -377,6 +378,17 @@ message CommitTxResponse {
     required TopoStatusCode statusCode = 1;
 }
 
+message GetCopysetInfoRequest {
+    required uint32 copysetId = 1;
+}
+
+message GetCopysetInfoResponse {
+    required TopoStatusCode statusCode = 1;
+    required metaserver.copyset.CopysetStatus copysetStatus = 2;
+    repeated common.PartitionInfo partitionInfoList = 3;
+    required uint32 inodeNumber = 4;
```

Comment:
The inodeNumber and dentryNumber will contained in common.PartitionInfo after heartbeat pr merged. The inode and dentry number of copyset can calculated when you get the partitionInfos of the copyset.

https://github.com/cw123/curve/blob/ba3b0b8264388f4f4c110af3d136585641bf4f08/curvefs/proto/common.proto#L59

Reply:
remove